docs/modules/edged.md

```diff
@@ -0,0 +1,61 @@
+# EdgeD
+
+## Overview
+
+EdgeD is an edge node module which manages the POD lifecycle. With help of the POD specifications provided as JSON from Cloud via MetaManager, it ensures the containers for the POD's (with containers as described in POD spec) are launched over the edge node, are running and healthy.
+
+EdgeD interfaces with MetaManager module to receive all the configuration calls. It responds to MetaManager module with all config results and queries.
+
+Docker container runtime is used for container and image management. It is the default option for EdgeD. It is a configurable option in config.yaml.
+
+Following functionalities are performed by edgeD :-
+
+- POD Launch, Deletion & Modification
+- ConfigMap Addition, Deletion & Modification
+- Secret Addition, Deletion & Modification
+- Probing Containers for Liveliness and Readiness
+
+## POD Launch, Deletion & Modification
+
+EdgeD supports Adding (Launching), Modification and Deletion of POD's. These requests are sent from Cloud, whenever respective "Kubectl" pod commands are execute over its terminal. The specs are defined in yaml format which is similar to that of Kubernetes. Those spec's can also have Volume and Network specs.
+
+If edgeD doesn't have the images in the spec, which shall be used to launch the pod, it will download it using docker runtime.
```

Comment:
All the details from this point onwards are not required. We can point to relevant K8S documentation for making users understand how to create pods, secrets, configmaps using kubectl commands in the previous section itself when we describe the responsibilities of edged.

Reply:
@rohitsardesai83 , i believe we need to come to a conclusion as @anvithks  has different opinion on it (as he say "If any section links to another site with the sample code, we should have atleast a basic version as an example on our site.").

Reply:
Try and keep the users within our documentation so that they don't have to look through different sites. Atleast a basic working example for our setup will help.  
@rohitsardesai83 What do you think? 
@m1093782566 Thoughts on this?